x-pack/legacy/plugins/maps/public/layers/styles/vector/vector_style.js

```diff
@@ -235,13 +219,11 @@ export class VectorStyle extends AbstractStyle {
       }
     };
 
-    scaledFields.forEach(({ min, max, name }) => {
-      if (min !== Infinity && max !== -Infinity) {
-        featuresMeta[name] = {
-          min,
-          max,
-          delta: max - min,
-        };
+    dynamicProperties.forEach(dynamicProperty => {
+      const styleMeta = dynamicProperty.pluckStyleMetaFromFeatures(features);
+      const name = dynamicProperty.getField().getName();
```

Comment:
You can move this line below the if condition since it's only needed if `styleMeta` is not null